src/app/app.component.html

```diff
@@ -48,7 +48,7 @@ <h2 class="d-none d-lg-block align-self-center col-6 text-body-emphasis" style="
       </ng-container>
 
         <li class="nav-item">
-          <a class="nav-link" onclick="window.open('https://cefirse.ird.fr')" href="">IndoFrench Cell</a>
+          <a class="nav-link" onclick="window.open('http://cefirse.ird.fr')" href="">IndoFrench Cell</a>
```

Comment:
Explain this

Reply:
Link was not functional, so changed it.

Reply:
http and https will not cause any issue in functionality. I have cross-verified the link is functional.
Using http is a security issue. Please revert this change